osbs/repo_utils.py

```diff
@@ -61,7 +62,10 @@ def __init__(self, dir_path='', file_name=REPO_CONFIG_FILE, depth=None):
         file_path = os.path.join(dir_path, REPO_CONTAINER_CONFIG)
         if os.path.exists(file_path):
             with open(file_path) as f:
-                self.container = (yaml.load(f) or {})
+                try:
+                    self.container = yaml.load(f) or {}
+                except yaml.scanner.ScannerError as e:
+                    raise OsbsException(str(e))
```

Comment:
Could be this err msg more descriptive, like "Failed to load YAML file {path}: {reason}"

Reply:
Respectively only related path `REPO_CONTAINER_CONFIG`, users don't care about our absolute/temp paths

Reply:
The reported error will look like this:
```
OsbsException('while scanning a simple key\n  in "/tmp/pytest-of-twaugh/pytest-3/test_create_build_invalid_yaml0/container.yaml", line 2, column 1\ncould not find expected \':\'\n  in "/tmp/pytest-of-twaugh/pytest-3/test_create_build_invalid_yaml0/container.yaml", line 2, column 4',)
```

I can't see a neat way to avoid repeating the absolute filename there, do you?

Reply:
Ah right, but still would be nice to see there that error is from YAML parsing

Reply:
`OsbsException('Failed to load YAML file containers.yaml: while scanning a simple key\n  in "/tmp/pytest-of-twaugh/pytest-3/test_create_build_invalid_yaml0/container.yaml", line 2, column 1\ncould not find expected \':\'\n  in "/tmp/pytest-of-twaugh/pytest-3/test_create_build_invalid_yaml0/container.yaml", line 2, column 4',)`